module-1/src/Components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,78 @@
+import { ReactNode } from "react";
+import { createPortal } from "react-dom";
+import "./Dialog.css";
+const mountElement: any = document.getElementById("overlays") || <div />;
+
+interface IDialogProps {
+  openDialog: boolean;
+  dialogTitle?: string;
+  children?: ReactNode;
```

Comment:
I would recommend not to add `children` to the components interface at all. It doesn't really make sense to type them nor can you do it properly.